webapp/channels/src/utils/text_formatting.test.ts

```diff
@@ -304,6 +306,101 @@ describe('highlightCurrentMentions', () => {
     });
 });
 
+describe('highlightWithoutNotificationKeywords', () => {
+    test('should replace highlight keywords with tokens', () => {
+        const text = 'This is a test message with some keywords';
+        const tokens = new Map();
+        const highlightKeys = [
+            {key: 'keywords'},
+            {key: 'test'},
+        ];
```

Comment:
Should we include phrases here with spaces as well for this test?

Reply:
added a test